src/Caching/Hybrid/src/Runtime/HybridCacheEntryFlags.cs

```diff
@@ -0,0 +1,50 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Microsoft.Extensions.Caching.Hybrid;
+
+/// <summary>
+/// Additional flags that apply to a <see cref="HybridCache"/> operation
+/// </summary>
+[Flags]
+public enum HybridCacheEntryFlags
+{
+    /// <summary>
+    /// No additional flags
+    /// </summary>
+    None = 0,
+    /// <summary>
+    /// Do not read from the local in-process cache
+    /// </summary>
+    DisableLocalCacheRead = 1 << 0,
+    /// <summary>
+    /// Do not write to the local in-process cache
+    /// </summary>
+    DisableLocalCacheWrite = 1 << 1,
```

Comment:
Why would you disable writing without also disabling reading?

Reply:
why do any of us do anything? I just don't think we need to bend the API to explicitly rule it out

Reply:
I'm not completely convinced, but I don't feel strongly about it.